operators/endpointmetrics/controllers/status/status_controller.go

```diff
@@ -9,22 +9,82 @@ import (
 	"fmt"
 	"net"
 	"reflect"
+	"sort"
 	"time"
 
+	"github.com/go-logr/logr"
```

Comment:
nit on the imports getting messed up here!

Reply:
what do you mean? Looks good to me.